### DISCUSSION TITLE
Reorder Buttons

### DISCUSSION THREAD

**Discussion by tmaleshafske** [TriliumNext/Trilium#1439] [Q&A]
Is it possible to Reorder the Buttons?

**Comment by jkaplon** [comment]
Well, for a temporary fix you can use your browser's dev tools to change the source order of the button elements.
Then you can be happy until your next page refresh puts everything back in original positions.
It's usually at that point that I decide not to care so much about the button ordering :-).

**Comment by zadam** [comment]
At this point no. But I do have plans to significantly change the way these buttons are set up and with the new way it will be possible to reorder them (and more).

However I'm not sure when this is coming ...
